### ISSUE TITLE
Hello, can I use it for multi label classification? If so, what should I pay attention to in the process of tag prediction? For multi label classification, sigmoid is generally used as the loss function. In this case, can you change your loss function to sigmoid?

### ISSUE THREAD

**Issue by ghost** [google-research/mixmatch#39]
Hello, can I use it for multi label classification? If so, what should I pay attention to in the process of tag prediction? For multi label classification, sigmoid is generally used as the loss function. In this case, can you change your loss function to sigmoid?

**Comment by david-berthelot** [comment]
We didn't MixMatch in this setting. One could see a multi-label classification loss as multiple binary losses to model the problem in a compatible way with MixMatch. I guess I can't really give more advice than that and experimenting with the code on your data.

**Comment by ghost** [comment]
Thank you for your reply. I used my own multi classification dataset (not multi label classification) to train mixmatch. The first few rounds of training worked well，after the first few epochs of network learning, the loss began to rise, the precision decreased, and many parameters were adjusted. Could this be due to the label prediction error of unlabeled samples, which led to the worse training effect of the model?

**Comment by david-berthelot** [comment]
It could be a lot of things, it's hard for me to tell. This repository is mostly to show how to reproduce the results achieved in the paper.
